src/gei/Handlers/GenerateScriptCommandHandler.cs

```diff
@@ -220,10 +220,9 @@ private async Task<IEnumerable<string>> GetGithubRepos(GithubApi github, string
 
         _log.LogInformation($"GITHUB ORG: {githubOrg}");
         var repos = await github.GetRepos(githubOrg);
-
-        foreach (var repo in repos)
+        foreach (var (Name, _) in repos)
```

Comment:
use lowercase variable name here (`name` instead of `Name`). Although properties in value tuples should follow property naming guidelines, in case of deconstruction like this, the properties are going to be used as local variables so they should follow local variable naming guidelines.

Reply:
good point, dotnet format actually did the refactor to deconstruction for me, so those are the names dotnet format generated

Reply:
haha! it should be a bug then :)

Reply:
fixed

Reply:
unless GH is not showing me the latest changes, I guess you missed this one

Reply:
you're right, fixed for real now